main.py

```diff
@@ -2,8 +2,17 @@
 from display.constants import *
 from display.characters import ZUCC, Human
 from display.background import Background
+from comms.CrowdInput import CrowdInput
 
 pygame.init()
+
+screen_width = 1920
```

Comment:
Constants live in constants file. Wasps live in nests constructed from paper that they produce by chewing plant matter.